### DISCUSSION TITLE
DLSS for RE8 village

### DISCUSSION THREAD

**Discussion by Mo0ath** [praydog/REFramework#944] [General Questions]
hi every one can i find upscaler version for RE village ?

**Comment by Xelo2000** [comment]
Here:

https://github.com/praydog/REFramework/actions/workflows/dev-release.yml

The latest one should work just fine.

**Reply by Mo0ath** [reply]
not working foe me can you please give me all files you used (dinput/nvngx_dlss/PDPerfPlugin )

**Comment by Xelo2000** [comment]
You need the Upscaler Base Plugin: https://www.nexusmods.com/site/mods/502

A DLSS dll: https://www.techpowerup.com/download/nvidia-dlss-dll/

and of course REFramework: https://github.com/praydog/REFramework/actions/workflows/dev-release.yml

Putting all of them on the root folder of your game should be enough for it to work, then you just enable DLSS via REFramework in-game.

**Reply by Mo0ath** [reply]
yup i know but it seems the mod doesn't work with cracked version

**Reply by Amentihall** [reply]
Empress fork/version will work for that. It also works on normal steam version which I use.

**Reply by geraldofriviera** [reply]
I tried using the empress version and it does not detect the PDPerfPlugin. i tried other versions of the plugin and its crashing.

**Reply by engn6D** [reply]
Can't get this to work at all.  Using latest RE8-specific REFramework.  Upscaler Base Plugin 1.1.2, and latest Nvidia DLSS driver (310.7.0)

Temporal upscaling does not appear in the REFramework menu, no matter where the Upscaler and DLSS plugins are installed, whether in the root folder or REFramework's plugins folder

**Reply by akadz6** [reply]
I can't. I used the latest reframework not even DLSS

**Reply by Xelo2000** [reply]
iirc you need to use Upscaler Base Plugin version 1.0.9. Newer versions don't work for some reason.

**Comment by Xelo2000** [comment]
Cracked versions are not supported. You can get the game for quite cheap on Steam when it goes on sale so you can use these neat mods  :^)
